docs/book/src/developer/providers/migrations/v1.4-to-v1.5.md

```diff
@@ -17,11 +17,11 @@ maintainers of providers and consumers of our Go API.
 
 ### Deprecation
 
--
+- API version `v1alpha3` is not served anymore from v1.5 (users can enable it manually in case they are lagging behind with deprecation cycles). Important: `v1alpha3` will be removed in 1.6.
```

Comment:
Not sure I understand why v1alpha3 is under deprecation and v1alpha4 under removal

Reply:
That would have been my general feeling :D